### DISCUSSION TITLE
The next alpha for Weather2 Remastered is coming soon

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#130] [Announcements]
Good afternoon everyone, it's the second week of the new year, and I am glad to announce that a playable alpha is close to completion! This alpha will be stable to run and will contain many new features to play around with.

This update will have half of the planned features, and will have bug fixes that came from 2.7.

It is expected to release sometime in Friday all the way up to March (Dependant on my pace). I hope you all enjoy it when it comes out!

**Comment by CosmicHikari** [comment]
**LOUD ECHOED APPLAUSE ACROSS THE ENTIRE SOLAR SYSTEM**

YAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA! Let's goooo
